app/models/placements/schools/placements/build/placement.rb

```diff
@@ -0,0 +1,66 @@
+class Placements::Schools::Placements::Build::Placement < Placement
+  has_many :placement_mentor_joins, dependent: :destroy
+  has_many :mentors, through: :placement_mentor_joins, class_name: "Placements::Mentor"
+
+  has_many :placement_subject_joins, dependent: :destroy
+  has_many :subjects, through: :placement_subject_joins
+
+  belongs_to :school, class_name: "Placements::School"
+
+  validates :school, :status, presence: true
+
+  attr_accessor :phase, :mentor_ids, :subject_ids
+
+  def valid_phase?
+    return true if phase.present? && [Placements::School::PRIMARY_PHASE, Placements::School::SECONDARY_PHASE].include?(phase)
```

Comment:
Not sure you need the `phase.present?` as `[Placements::School::PRIMARY_PHASE, Placements::School::SECONDARY_PHASE].include?(nil)` will return false